apiconfig/prodApiConfig.js

```diff
@@ -1,3 +1,5 @@
 module.exports = {
   kathakaliCharacterClassificationApi: 'http://char-recognition:5000/classify',
+  facialDetectionApi: 'http://facial-detection:5000/detect',
+  expressionDetectionApi: 'http://localhost:5000/expression',
```

Comment:
@nzixuan not sure why it kept defaulting to facial-expression although I am in DEV, is there any other settings that I should be doing?

Reply:
for some reason npm run start is running in dev mode i added a run dev command in the new pr
https://github.com/culture-web/culture-web-be/pull/23/files 

also ur expression detection API link in prod needs to be some name besides localhost cause it will be the container name in the deployment